Drv/LinuxGpioDriver/LinuxGpioDriver.hpp

```diff
@@ -1,17 +1,140 @@
 // ======================================================================
-// LinuxGpioDriver.hpp
-// Standardization header for LinuxGpioDriver
+// \title  LinuxGpioDriver.hpp
+// \author lestarch
+// \brief  hpp file for LinuxGpioDriver component implementation class
+//
+// \copyright
+// Copyright 2009-2015, by the California Institute of Technology.
+// ALL RIGHTS RESERVED.  United States Government Sponsorship
+// acknowledged.
+//
 // ======================================================================
 
-#ifndef Drv_LinuxGpioDriver_HPP
-#define Drv_LinuxGpioDriver_HPP
+#ifndef LinuxGpioDriver_HPP
```

Comment:
Might want to qualify the include guard just in case `Drv_LinuxGpioDriver_HPP`

Reply:
Agreed.